### PR TITLE
feat: [SRM-11182]: UX review comments

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary.  UX changes as per https://harness.atlassian.net/browse/SRM-11182

Updated Title container for Templates card and Template studio  from SVG to  Styled component

#### Screenshots
Screenshot 2022-07-26 at 12 21 12 PM
Screenshot 2022-07-26 at 12 20 09 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx [hunk 1]
```diff
@@ -92,6 +93,18 @@ export interface ConfigModalProps extends ModalProps {
   onClose: () => void
 }
 
+export const getTemplateTitleFromType = (
```
Comment (deepesh-ui) [inline src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx:96]: Inside **Create New Template**  Modal, we don't want  to populate the default new template name rather we are keeping it Empty.

**Review by yc147 — COMMENTED** [review]
When creating a new template, you have removed the default value for the name field i.e. New {entity} Template Name in case of monitored service template. If this is a real use case then let’s solve it in a way that in future other templates can also benefit.

Also, in case of monitored service template you are getting template type label from a separate string in the codebase instead of using template.type.

**Review by yc147 — CHANGES_REQUESTED** [review]
When creating a new template, you have removed the default value for the name field i.e. New {entity} Template Name in case of monitored service template. If this is a real use case then let’s solve it in a way that in future other templates can also benefit.

Also, in case of monitored service template you are getting template type label from a separate string in the codebase instead of using template.type.

**Review by yc147 — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Comment by deepesh-ui** [comment]
Coverage failure is due to newly added templates , which don't have any functionality only classes are added for future reference

Screenshot 2022-07-26 at 4 09 28 PM

**Pull request merged**